buildSrc/src/main/groovy/Docker.groovy

```diff
@@ -249,9 +249,13 @@ class Docker {
             sync.into dockerWorkspaceContents
 
             if (cfg.dockerfileFile) {
-                sync.from cfg.dockerfileFile
+                sync.from(cfg.dockerfileFile) { CopySpec dockerfileCopy ->
+                    dockerfileCopy.include('Dockerfile')
```

Comment:
Should we take the provided file name instead of assuming Dockerfile?

Reply:
Yes, that's a good idea, I'll correct that.